perf/istio/gateway-bouncer/setup.sh

```diff
@@ -0,0 +1,46 @@
+#/bin/bash
+set -ex
+
+NAMESPACE=${NAMESPACE:?"New/existing NAMESPACE to install this scenario into"}
+
+function install_gateway_bouncer() {
+  local DIRNAME="${1:?"output dir"}"
+  local OUTFILE="${DIRNAME}/gateway_bouncer.yaml"
+  local INGRESS_IP=""
+
+  helm -n $NAMESPACE template \
+    --set namespace=$NAMESPACE \
+    . > "${OUTFILE}"
+
+  if [[ -z "${DRY_RUN}" ]]; then
+    kubectl -n $NAMESPACE apply -f "${OUTFILE}"
+
+    # Waiting until LoadBalancer is created and retrieving the assigned
+    # external IP address.
+    while : ; do
+      INGRESS_IP=$(kubectl -n $NAMESPACE \
+        get service istio-ingress-$NAMESPACE \
+        -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+      if [[ -z "${INGRESS_IP}" ]]; then
+        sleep 5s
+      else
+        break
+      fi
+    done
+
+    # Populating a ConfigMap with the external IP address and restarting the
+    # client to pick up the new version of the ConfigMap.
+    kubectl -n $NAMESPACE delete configmap fortio-client-config
+    kubectl -n $NAMESPACE create configmap fortio-client-config \
+      --from-literal=external_addr=$INGRESS_IP
+    kubectl -n $NAMESPACE patch deployment istio-ingress-$NAMESPACE \
+      -p "{\"spec\":{\"template\":{\"metadata\":{\"labels\":{\"date\":\"`date +'%s'`\"}}}}}"
```

Comment:
Generally I prefer patching an annotation, not label ( labels create secondary indexes and trigger a lot of other side effects )